x/sequencer/keeper/invariants.go

```diff
@@ -0,0 +1,112 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
+)
+
+// RegisterInvariants registers the bank module invariants
+func RegisterInvariants(ir sdk.InvariantRegistry, k Keeper) {
+	ir.RegisterRoute(types.ModuleName, "sequencers-count", SequencersCountInvariant(k))
+	ir.RegisterRoute(types.ModuleName, "sequencers-per-rollapp", SequencersPerRollappInvariant(k))
+}
+
+// AllInvariants runs all invariants of the X/bank module.
+func AllInvariants(k Keeper) sdk.Invariant {
+	return func(ctx sdk.Context) (string, bool) {
+		res, stop := SequencersCountInvariant(k)(ctx)
+		if stop {
+			return res, stop
+		}
+		res, stop = SequencersPerRollappInvariant(k)(ctx)
+		if stop {
+			return res, stop
+		}
+		return "", false
+	}
+}
+
+func SequencersCountInvariant(k Keeper) sdk.Invariant {
+	return func(ctx sdk.Context) (string, bool) {
+		var (
+			broken bool
+			msg    string
+		)
+
+		sequencers := k.GetAllSequencers(ctx)
+		rollapps := k.rollappKeeper.GetAllRollapp(ctx)
+
+		totalCount := 0
+		for _, rollapp := range rollapps {
+			seqByRollapp := k.GetSequencersByRollapp(ctx, rollapp.RollappId)
+			bonded := k.GetSequencersByRollappByStatus(ctx, rollapp.RollappId, types.Bonded)
```

Comment:
what about the unspecified status? not sure if it's used at all but if not we can just remove it all together.

Reply:
not used, and not expected to be in store